traffic_router/core/src/main/java/com/comcast/cdn/traffic_control/traffic_router/core/router/TrafficRouter.java

```diff
@@ -846,11 +888,39 @@ protected CacheLocation getCoverageZoneCacheLocation(final String ip, final Deli
 		return getCoverageZoneCacheLocation(ip, deliveryService.getId());
 	}
 
+	/**
+	 * Chooses a {@link Cache} for a Delivery Service based on the Coverage Zone File given a clients IP and request *path*.
+	 *
+	 * @param ip The client's IP address
+	 * @param deliveryServiceId The "xml_id" of a Delivery Service being routed
+	 * @param requestPath The client's requested path - e.g. 'http://test.example.com/request/path' -> '/request/path'
+	 * @return A cache object chosen to serve the client's request
+	 */
 	public Cache consistentHashForCoverageZone(final String ip, final String deliveryServiceId, final String requestPath) {
 		return consistentHashForCoverageZone(ip, deliveryServiceId, requestPath, false);
 	}
 
+	/**
+	 * Chooses a {@link Cache} for a Delivery Service based on the Coverage Zone File or Deep Coverage Zone File given a clients IP and request *path*.
+	 *
+	 * @param ip The client's IP address
+	 * @param deliveryServiceId The "xml_id" of a Delivery Service being routed
+	 * @param requestPath The client's requested path - e.g. 'http://test.example.com/request/path' -> '/request/path'
+	 * @param useDeep if `true` will attempt to use Deep Coverage Zones - otherwise will only use Coverage Zone File
+	 * @return A cache object chosen to serve the client's request
+	 */
 	public Cache consistentHashForCoverageZone(final String ip, final String deliveryServiceId, final String requestPath, final boolean useDeep) {
+		final HTTPRequest r = new HTTPRequest();
+		r.setPath(requestPath);
+		r.setQueryString("");
+		return consistentHashForCoverageZone(ip, deliveryServiceId, r, useDeep);
+	}
+
+	public Cache consistentHashForCoverageZone(final String ip, final String deliveryServiceId, final HTTPRequest request) {
```

Comment:
My IDE is flagging this method as never used. Can it be deleted?

Reply:
but it's used literally 3 lines above its declaration...

Reply:
nvm - this actually is getting bypassed. that other one though...